src/core/java/com/enderio/core/client/rendering/Animation.java

```diff
@@ -0,0 +1,44 @@
+package com.enderio.core.client.rendering;
+
+import net.minecraft.util.Mth;
+
+public class Animation {
```

Comment:
Javadocs please :)

Reply:
Fixed in cb65ecffc869a1b00025ff0c587219e4f69e6153